guides/hack/21-XHP/01-setup.md

```diff
@@ -1,13 +1,41 @@
-## Namespace Support
-
-Support for namespaced XHP classes (elements) is enabled by default since
+Support for XHP and namespaced XHP classes (elements like `<p>`) is enabled by default since
```

Comment:
Hhvm has always supported xhp and it is unconditionally enable. 4.73 made no mangling mode the default.

Reply:
yes, my mistake. reverting.